packages/elements/src/slider/elements/slider.ts

```diff
@@ -561,6 +590,35 @@ export class Slider extends ControlElement {
     }
   }
 
+  /**
+   * Update the ARIA value text for a given thumb.
+   *
+   * @param thumbRef - The reference to the thumb element.
+   * @param markerValue - The value associated with the marker.
+   * @returns {void}
+   */
+  private updateAriaValueText(thumbRef: HTMLDivElement | undefined, markerValue: string): void {
```

Comment:
passing Element Reference instead of HTMLElement into the function is cleaner.
```suggestion
  private updateAriaValueText(thumbRef: Ref<HTMLDivElement>, markerValue: string): void {
```